frontend/src/types/index.ts

```diff
@@ -219,3 +220,82 @@ export interface TransactionsInitialState {
   status: ActionStatus | undefined;
   errorString?: string;
 }
+
+export interface OperationsInitialState {
+  lastOperations: FetchLastOperationsActionResponse[];
+  status: ActionStatus | undefined;
+  errorString?: string;
+}
+
+export interface OperationsResponse {
```

Comment:
is there a reason we're not grabbing this type from the stellar js sdk? The response type should def be in there

Reply:
now I fixed it to call the SDK data, but the response is not typed